src/test/java/core/basesyntax/dao/FruitTransactionDaoImplTest.java

```diff
@@ -0,0 +1,63 @@
+package core.basesyntax.dao;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertNull;
+import static org.junit.Assert.assertThrows;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.model.Operation;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+class FruitTransactionDaoImplTest {
+    private static final String APPLE = "apple";
+    private static FruitTransactionDao fruitTransactionDao;
+
+    @BeforeAll
+    static void beforeAll() {
+        fruitTransactionDao = new FruitTransactionDaoImpl();
+    }
+
+    @Test
+    void get_FromStorage_isOk() {
```

Comment:
First letter after _ should be lower case.

Reply:
Here you test get and store in the same test.
Separate to two tests:

- store with Storage and get with fruitTransactionDao
- store with fruitTransactionDao and get with Storage